builds/azure-pipelines/template-steps-build-test-linux.yml

```diff
@@ -6,7 +6,12 @@ parameters:
 
 steps:
 - task: UseDotNet@2
-  displayName: 'Install .NET Core SDK 3.1.x for Build'
+  displayName: 'Install .NET Core SDK 5.0.x for Build'
```

Comment:
are we really using this for the build? Or just for running the tests?

Reply:
Should be for tests - the Microsoft.NET.Test.Sdk v17 did not work with .NET Core SDK v3.1